src/services/annotation/service/routes.mjs

```diff
@@ -47,16 +51,40 @@ export const routes = (fastify, options, done) => {
 		const { bucket: inBucket, key: inKey } = parseS3URI(s3_input_uri);
 		const { bucket: outBucket, key: outKey } = parseS3URI(s3_output_uri);
 
-		const index = uriToEsIndex(s3_input_uri);
+		const { email } = parseBasicAuth(request.headers.authorization);
+
+		// const index = uriToEsIndex(s3_input_uri);
+		const index = id;
 		await bucketToIndex(
 			index,
 			domain,
 			inBucket,
 			inKey,
+			idField
 		);
```

Comment:
(1) probably for a refactor PR, but I've thinking I'd prefer passing a single object rather than multiple args, so that we're not depending on the args order ant as a benefit we can sort them alphabetically whcih makes code more consistent

Reply:
I suggest we leave it for a refactor PR, as it will change many interconnected pieces of logic if we change the method signature.